contracts/test/ERC20Mock.sol

```diff
@@ -6,7 +6,7 @@ import "openzeppelin-contracts-sol8/token/ERC20/ERC20.sol";
 
 contract ERC20Mock is ERC20 {
 
-    uint256 constant public initialSupply = 20000000000000000000000;
+    uint256 constant public initialSupply = 200000000000000000000000000;
```

Comment:
For my own understanding. Why is the initialSupply of the Mock increased in the cleanup?

Reply:
Thanks for spotting. This is actually not necessary, I used it to test something, but forgot to remove it.